chess-AI/train.py

```diff
@@ -16,6 +13,33 @@
 from stockfish_train import StockfishTrain
 
 
+class TrainOptions:
+    """Stores settings for the training episodes
+
+        Attributes:
+            learning_rate: the learning rate for the optimizer
+            momentum: additional parameter for the optimizer
+            weight_decay: loss term encouraging smaller weights
+            epochs: number of epochs
+            batch_size: the batch size for SGD
+            games: number of games to run when creating dataset
+            device: the device being used to train (either CPU or GPU)
+            save_path: path for model checkpointing
+            num_saved_models: number of models to store
+        """
```

Comment:
This entire block should be indented one level less, so that the start and end """ line up